crawler/file_processing.py

```diff
@@ -398,7 +397,7 @@ def set_state_for_file(self) -> CentreFileState:
             self.file_state = CentreFileState.FILE_PROCESSED_WITH_SUCCESS
 
         # check for this being an unconsolidated samples file where the centre doesn't support those
-        elif centre.get(CENTRES_KEY_SKIP_UNCONSOLIDATED_FILES, False) and self.is_unconsolidated_surveillance_file():
+        elif centre.get("skip_unconsolidated_surveillance_files", False) and self.is_unconsolidated_surveillance_file():
```

Comment:
Why don't we use it as a constant anymore?

Reply:
It was to improve the typing checks. We could extract them but then they are duplicated in a way.

Reply:
I found a way to fix it.